crates/biome_css_analyze/src/keywords.rs

```diff
@@ -855,13 +855,73 @@ pub const OTHER_PSEUDO_ELEMENTS: [&str; 18] = [
     "view-transition-old",
 ];
 
-pub const VENDER_PREFIXES: [&str; 4] = ["-webkit-", "-moz-", "-ms-", "-o-"];
+pub const MEDIA_FEATURE_NAMES: [&str; 57] = [
+    "any-hover",
+    "any-pointer",
+    "aspect-ratio",
+    "color",
+    "color-gamut",
+    "color-index",
+    "device-aspect-ratio",
+    "device-height",
+    "device-width",
+    "display-mode",
+    "dynamic-range",
+    "environment-blending",
+    "forced-colors",
+    "grid",
+    "height",
+    "horizontal-viewport-segments",
+    "hover",
+    "inverted-colors",
```

Comment:
`light-level` is missing.

ref: https://daveordead.github.io/light-levels-demo/
tested on chrome v111 and Firefox v115